scrapli/transport/base/base_transport.py

```diff
@@ -23,6 +23,8 @@ class BasePluginTransportArgs:
 
 
 class BaseTransport(ABC):
+    SSH_SYSTEM_CONFIG_MAGIC_STRING: str = "__SCRAPLI_RULES__"
```

Comment:
lol, while I love and appreciate this placeholder string... lets do two things:

1) I think this is really only ever going to be relevant for system transport right? if "yes", maybe we should put the it in the system transport package?
2) hah scrapli does rule, but maybe we call it something more descriptive for the variable and the actual string -- like SYSTEM_TRANSPORT_SSH_CONFIG_TRUE: str = "__SYSTEM_TRANSPORT_SSH_CONFIG_TRUE__" lol so much less fun but probably more understandable :P

Reply:
Yeah probably not the best place to drop a compliment, but hey :laughing:  I changed it to what you suggested!